src/lib/api/app.ts

```diff
@@ -65,7 +78,7 @@ export default async function( app: string | number, fields: ?any, fragments: ?s
 			},
 		} );
 
-	if ( ! res || ! res.data || ! res.data.app ) {
+	if ( ! res.data.app ) {
```

Comment:
```suggestion
	if ( ! res?.data?.app ) {
```
Do we need to still check individual parts?

Reply:
```ts
query<T = any, TVariables = OperationVariables>(options: QueryOptions<TVariables, T>): Promise<ApolloQueryResult<T>>;

export declare type ApolloQueryResult<T> = {
    data: T;
    errors?: ReadonlyArray<GraphQLError>;
    error?: ApolloError;
    loading: boolean;
    networkStatus: NetworkStatus;
    partial?: boolean;
};
```

`res` is truthy because `ApolloQueryResult` is truthy;
`res.data` is also truthy because `data` are always there.